### PR TITLE
Make the 'avr-std-stub' dependency optional

### PR DESCRIPTION
**Pull request by dylanmckay**
This way, downstream users may user their own panic handlers and
exception handling personality functions if they desire.

### CODE REVIEW

**Comment by shepmaster** [comment]
🤣 #26

File: src/lib.rs [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 #![no_std]
 
-extern crate avr_std_stub;
+#[cfg(feature = "avr-std-stub")] extern crate avr_std_stub;
```
Comment (shepmaster) [inline src/lib.rs:12]: Why keep this at all?

Comment (dylanmckay) [reply src/lib.rs:12]: The `extern crate` statement?

Comment (dylanmckay) [reply src/lib.rs:12]: I found that `extern crate` is required - it is not enough to declare it in Cargo.toml. If the extern crate is missing, it doesn't pick up the panic handler or the eh personality from the upstream crate - it is kinda weird

**Pull request merged**